### ISSUE TITLE
Agenda going forward or back daily and not monthly

### ISSUE THREAD

**Issue by Zanther** [intljusticemission/react-big-calendar#755]
I just come onto this project from a previous dev and he has set up big calendar to show one month of events, but when the user clicks on next or previous it only goes ahead or back by one day. How do I change it back to go forward or previous 30 days? 

I looked everywhere in the code and I cannot find the option,  I found AgendaWeek and day set to one.  I set it to 4 weeks / 30 days respectively and still has not changed.

**Comment by dabo02** [comment]
I have this same problem. Also, neither the agenda nor the month view renders. Day and week work fine but buggy with the problem that @Zanther describes

**Comment by STEVEOO6** [comment]
@Zanther I had similar requirements. I ended up creating a custom view modelled off Agenda. I used startOf and endOf month calculations throughout to get the outcome I wanted e.g.
`
import dateMath from 'date-arithmetic'
...
const startOfMonth = dateMath.startOf(date, 'month')
const endOfMonth = dateMath.endOf(date, 'month')
let range = dates.range(start, end, 'day')
events = events.filter(event => inRange(event, start, end, this.props))
`
